templates/ocs-deployment/rbd_storage_class.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: csi-rbd
+provisioner: rbd.csi.ceph.com
+parameters:
+    # Comma separated list of Ceph monitors
+    # if using FQDN, make sure csi plugin's dns policy is appropriate.
+    monitors: {{ mon_ports }}
+
+    # if "monitors" parameter is not set, driver to get monitors from same
+    # secret as admin/user credentials. "monValueFromSecret" provides the
+    # key in the secret whose value is the mons
+    #monValueFromSecret: "monitors"
+    
+    # Ceph pool into which the RBD image shall be created
+    pool: {{ pool_name }}
+
+    # RBD image format. Defaults to "2".
+    imageFormat: "2"
+
+    # RBD image features. Available for imageFormat: "2". CSI RBD currently supports only `layering` feature.
+    imageFeatures: layering
+    
+    # The secrets have to contain Ceph admin credentials.
+    csi.storage.k8s.io/provisioner-secret-name: {{ ceph_secret }}
+    csi.storage.k8s.io/provisioner-secret-namespace: default
```

Comment:
The yaml is expecting a secret which was created in default namespace
e.g., 
#oc get secret -n default |grep csi
csi-rbd-secret

This secret is supposedly created as part of oc create -f [secret.yaml](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/example/rbd/secret.yaml)